math/find_number_of_iteration_to_kaprekar_constant.cpp

```diff
@@ -0,0 +1,80 @@
+/**
+ * @author [jasonchee816](https://github.com/jasonchee816)
+ * @file
+ *
+ * @brief [Program to count iterations needed to reach Kaprekar's Constant, 6174

```

Comment:
Could you please add a detailed description of how the algorithm works, what it does, etc.?
Let us know if you need any help providing this! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix